src/jvm/test/org/jetbrains/kotlinx/lincheck_test/representation/SuspendTraceReportingTest.kt

```diff
@@ -17,6 +17,7 @@ import org.junit.*
 import kotlin.coroutines.*
 import kotlinx.coroutines.sync.*
 
+@IgnoreInTraceDebuggerMode // TODO investigate (Evgeniy Moiseenko)
```

Comment:
Why is this test affected?

Reply:
Because of the reasons mentioned above by @eupp.